lib/twitter_cldr/js/renderers/numbers/rbnf/rbnf_test_helper_renderer.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: UTF-8
+
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+module TwitterCldr
+  module Js
+    module Renderers
+      module TestHelpers
+
+        class RBNFTestHelperRenderer < TwitterCldr::Js::Renderers::Base
+          set_template "mustache/numbers/rbnf/rbnf_test_helper.coffee"
+
+          def global_resource
+            result = {}
+            TwitterCldr.supported_locales.each do |locale|
+              result[locale] = TwitterCldr.resources.get_locale_resource(locale, "rbnf")[locale][:rbnf][:grouping]
+            end
+            result.to_json
+          end
+
+          def test_resource
+            result = {}
+            TwitterCldr.supported_locales.each do |locale|
```

Comment:
Same here – use `inject` so that you don't need to initialize the container separately.